task/syft/0.1/tests/run-daemon.yaml

```
@@ -0,0 +1,32 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: pipeline-syft-test
+spec:
+  params:
+    - name: ARGS
+      type: array
+      description: The Arguments to be passed to Syft command for config image.
+  description: Test Pipeline for Syft
+  tasks:
+    - name: syft-image-sbom
+      taskRef:
+        name: syft
+        kind: Task
+      params:
+        - name: ARGS
+          value: ["$(params.ARGS[*])"]
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: pipelinerun-syft-test-
+spec:
+  params:
+    - name: ARGS
+      value:
+        - "docker.io/devopps/hello-world-cli"
```

We could scan a tekton image

```suggestion
          value: gcr.io/tekton-releases/dogfooding/tkn
```

```suggestion
          value: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/webhook
```

or the grype image itself

```suggestion
          value: docker.io/anchore/grype
```

rather than depending on another image that can change underneath us

---

One issue with my comment here is if one of these has a vuln the test would fail. There is a `--fail-on` flag where you can increase the failure threshold but there's not a flag to just not fail. And I can't find a way to allow a task within a pipeline to fail just yet (but there is a TEP)

Maybe we just build a tiny container and scan it locally like some of the other tests that build containers. This could have the benefit of not requiring external resources